arches/app/media/js/views/components/widgets/rich-text.js

```diff
@@ -29,6 +29,7 @@ define([
         const currentLanguage = {"code": arches.activeLanguage};
         let currentValue = koMapping.toJS(self.value) || initialCurrent;
         self.currentLanguage = ko.observable(currentLanguage);
+        self.currentPlaceholder = ko.observable();
```

Comment:
The crux of this solution is to just treat the placeholder field like a genuine localized field and ensure we're never using shortcuts that immediately unwrap to a certain language.

Reply:
... er, only use shortcuts in the "presentation layer" of the template without altering card truth, which is why this variable uses "current" in its name